files/phoneformat.js/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "phoneformat.js",
+  "repo": "albeebe/phoneformat.js",
+  "files": {
+    "basePath": "dist",
+    "include": ["phone-format.js", "phone-format.min.js", "phone-format.min.js.map"]
```

Comment:
I'm sorry, but I'm not fluent in git, and really lost my way through rebasing, merging and pushing stuff between several local and remote branches, without any idea how the pull request is affected by it (I've used the github web-interface for this PR).
Isn't there any other way to input these parameters to the download-bot?